### PR TITLE
Added code to add label to reference organ

### PR DESCRIPTION
**Pull request by andreasbueckle**
@bherr2 
I would like to add a column with the ```organ_label``` synonymous to what we did here: https://github.com/cns-iu/hra-cell-type-populations-supporting-information/blob/main/data-processor/queries/as-datasets-modality.rq#L45-L46. This query, when run, returns empty results. Could you please review? Thanks!

### CODE REVIEW

**Comment by bherr2** [comment]
Sorry, I don't understand what your're asking. Is it as-datasets-modality.rq that doesn't  work or another one?

File: data-processor/queries/table-s5.rq [hunk 1]
```diff
@@ -36,8 +39,16 @@ WHERE {
 					ccf:has_reference_organ ?ref_organ ;
 					ccf:percentage_of_total ?percentage ;
     	]
-    ]
+    ] .
+
+	# get label for reference organ
+	?refOrgan ccf:representation_of ?organId .
```
Comment (bherr2) [inline data-processor/queries/table-s5.rq:45]: ?refOrgan => ?ref_organ

**Comment by bherr2** [comment]
Ok, I commented on the  changes. You have `?refOrgan` but `?ref_organ` is the variable used in the query

File: data-processor/queries/table-s5.rq (hunk 1 above)
Comment (andreasbueckle) [reply data-processor/queries/table-s5.rq]: Thanks, that did it!

**Pull request merged**